src/__tests__/__snapshots__/snapshots.test.js

```diff
@@ -0,0 +1,91 @@
+import TestRenderer from 'react-test-renderer';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from '../../redux/configureStore';
+import Crypto from '../../components/Crypto';
+import EFTs from '../../components/EFTs';
+import Home from '../../components/Home';
+import Forex from '../../components/Forex';
+import Loading from '../../components/Loading';
+import Majors from '../../components/Majors';
+import Stocks from '../../components/Stocks';
+
+describe('test screenshots for all components', () => {
+  test('matches Crypto snapshot', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
+        <Router>
+          <Crypto />
+        </Router>
+      </Provider>,
+    ).toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('matches Home snapshot', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
+        <Router>
+          <Home />
+        </Router>
+      </Provider>,
+    ).toJSON();
+
+    expect(tree).toMatchSnapshot();
+  });
+
+  test('matches EFTs snapshot', () => {
+    const tree = TestRenderer.create(
+      <Provider store={store}>
```

Comment:
- [ ] some of your tests are failing, kindly fix this
![image](https://user-images.githubusercontent.com/66097371/227646705-54ee4a18-a01a-48e6-9c46-15f93811950c.png)

Reply:
@VICTOR-CHUKWU, 
tests fixed, 
Thank you for your review